packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -50,281 +51,310 @@ export const Default = {
 };
 
 /**
- * The sd-flipcard can be displayed in several ways using the `front-variant` and `back-variant` attributes. This example shows the usage `front-variant` attribute.
+ * Use the `variant` attribute to set the color variant for front or back.
+ * - empty (default)
+ * - primary
+ * - neutral-100
+ * - gradient-light-top
+ * - gradient-light-bottom
+ * - gradient-dark-top
+ * - gradient-dark-bottom
  */
 
 export const Variants = {
-  parameters: { controls: { exclude: ['front-variant'] } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        y: {
-          type: 'attribute',
-          name: 'front-variant'
-        }
-      },
-      args,
-      constants: [
-        {
-          type: 'template',
-          name: 'style',
-          value: '<div style="margin-bottom: 40px">%TEMPLATE%</div>'
-        }
-      ]
-    })
-};
-
-/**
- * Use the `activation` attribute to determine the activation type of the flipcard. There are two options: `click-only` and `hover-and-click`.
- */
-
-export const Activation = {
-  parameters: { controls: { exclude: ['activation'] } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'activation'
-        }
-      },
-      args,
-      constants: [
-        {
-          type: 'template',
-          name: 'style',
-          value: '<div style="margin-bottom: 40px">%TEMPLATE%</div>'
-        }
-      ]
-    })
-};
-
-/**
- * Use the `flip-direction` attribute to determine the direction of the flipcard. There are two options: `horizontal` and `vertical`.
- */
-
-export const flipDirection = {
-  parameters: { controls: { exclude: ['flip-direction'] } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'flip-direction'
-        }
-      },
-      args,
-      constants: [
-        {
-          type: 'template',
-          name: 'style',
-          value: '<div style="margin-bottom: 40px">%TEMPLATE%</div>'
-        }
-      ]
-    })
+  render: () =>
+    html` <div class="grid grid-cols-2">
+      <div style="margin-bottom: 40px">
```

Comment:
Please use classes instead of inline styles.
Also, I think you can remove the divs around the flipcards and set the `gap-*` in the parent wrapper.

Reply:
Updated